sippy-ng/src/component_readiness/CompReadyCapsCell.js

```diff
@@ -29,6 +30,25 @@ export default function CompReadyCapsCell(props) {
   const { status, columnVal, capabilityName, filterVals } = props
   const theme = useTheme()
 
+  const [capabilityParam, setCapabilityParam] = useQueryParam(
+    'capability',
+    StringParam
+  )
+  const [environmentParam, setEnvironmentParam] = useQueryParam(
+    'environment',
+    StringParam
+  )
+
+  const handleClick = (event) => {
+    event.preventDefault()
+    setCapabilityParam(capabilityName)
+    setEnvironmentParam(columnVal)
+    window.location.href = capabilityReport(
+      capabilityName,
+      columnVal,
+      filterVals
+    )
+  }
```

Comment:
This strikes me as a hack that's working around a real problem we need to figure out